src/components/ec-input-field/ec-input-field.vue

```diff
@@ -26,6 +26,8 @@
       class="ec-input-field__input"
       :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-input-field__input` : 'ec-input-field__input'"
       :class="{
+        'ec-input-field__input--is-in-group-right': isInGroup === 'right' ? true : false,
```

Comment:
isn't this redundant? if true then true?

Reply:
well, you can write both statement into single line:
```
`ec-input-field__input--is-in-group-${isInGroup}`: !!isInGroup,
```